docs/ruleset_functions/get_proposed_hvac_modeled_with_virtual_cooling.md

```diff
@@ -0,0 +1,40 @@
+# get_proposed_hvac_modeled_with_virtual_cooling
+
+**Description:** Get the list of HeatingVentilationAirAconditioningSystem in which Appendix G Table G3.1 #10 d is applicable.
+
```

Comment:
@claperle Might be helpful to add the explicit rule here, i.e. Where no cooling system exists or no cooling system has
been submitted with design documents, the cooling system
type shall be the same as modeled in the baseline building
design and shall comply with the requirements of Section 6.
Also should clarify that this is a requirement for the proposed building.

If possible, I it would be helpful to provide a description of the App G requirement wherever we mention the section number.